src/Security.php

```diff
@@ -24,296 +26,50 @@
  * @author Derek Jones <derek.jones@ellislab.com>
  * @author Graham Campbell <graham@alt-three.com>
  */
-class Security
+final class Security
```

Comment:
Please don't make this class final, so it can be mocked easily. I am not particularly interested in adding an interface so we can make this class final.

Reply:
removed